tuf/api/metadata.py

```diff
@@ -534,22 +534,41 @@ def __init__(
         super().__init__(_type, version, spec_version, expires)
         self.meta = meta
 
+    @classmethod
+    def _get_snapshot_filename(cls, meta_dict):
```

Comment:
This is a reasonable check but aren't we just assuming that input data is correct everywhere else? Since input validation is still a WIP? Anyway, probably better having it than not.

Reply:
Well, the problem is that `meta_dict` sometimes is passed from the user - when calling `Timestamp.from_dict()`
and consequently when calling the `cls. _get_snapshot_filename()` method we haven't validated the `meta_dict` field.

At least, that is the case before implementing any new validation option as described in ADR 007.